backend/proxy/default.conf

```diff
@@ -1,14 +1,32 @@
-server {
-  listen $PORT;
-
-  location /v2 {
-    rewrite  ^/v2(/.*) $1 break;
-    proxy_pass https://$BACKEND_V2_HOSTNAME;
-    proxy_ssl_server_name on;
-  }
-
-  location / {
-    proxy_pass https://$BACKEND_V1_HOSTNAME;
-    proxy_ssl_server_name on;
-  }
+events {}
+
+http {
+    proxy_cache_path /tmp/cache_nginx/ levels=1:2 keys_zone=webapp_cache:10m max_size=10g inactive=1440 use_temp_path=off;
```

Comment:
can we set inactive to `24h`, I think it's in seconds by default, so 1440 would be 24 minutes.